test/Microsoft.Framework.Configuration.Binder.Test/project.json

```diff
@@ -1,14 +1,15 @@
-﻿{
-  "dependencies": {
-    "Microsoft.Framework.Configuration.Binder": "1.0.0-*",
-    "Microsoft.Framework.Configuration.Test.Common": "1.0.0-*",
-    "xunit.runner.aspnet": "2.0.0-aspnet-*"
-  },
-  "frameworks": {
-    "dnx451": { },
-    "dnxcore50": { }
-  },
-  "commands": {
-    "test": "xunit.runner.aspnet"
-  }
-}
+﻿{
+  "dependencies": {
+    "Microsoft.AspNet.Testing": "1.0.0-*",

```

Comment:
"Microsoft.AspNet.Testing": "1.0.0-*", was added
for some reason, it treats this as full delete/add, don#t know why, tried it a few times though
No spacing didn't change